apps/next/components/Register/index.tsx

```diff
@@ -33,27 +33,41 @@ export enum RegisterState {
   Registered, // registration complete (count => 3)
 }
 
-const Alert = ({ available }: { available: boolean }) => {
+const Alert = ({ available, registerState }: { available: boolean; registerState:  RegisterState | undefined }) => {
```

Comment:
Is the register state undefined to show loading animations?

Reply:
Yes